sdks/java/io/debezium/build.gradle

```diff
@@ -18,6 +18,18 @@
 import groovy.json.JsonOutput
 
 plugins { id 'org.apache.beam.module' }
+
```

Comment:
We should generalize this logic and the following and put them into BeamModulePlugin, in "applyJavaNature", consider add an optional parameter "requireJavaVersion". If set to 17, it will try to get java17Home and java21Home.